integration/bip0009_test.go

```diff
@@ -299,6 +299,7 @@ func TestBIP0009(t *testing.T) {
 	t.Parallel()
 
 	testBIP0009(t, "dummy", chaincfg.DeploymentTestDummy)
+	testSegwit(t, "segwit", chaincfg.DeploymentSegwit)
```

Comment:
Hmm, must've snuck in doing a borked rebase (and forgot to set the proper build flag to run the integration tests). Fixed!